test/integration/patients/sidebar.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'underscore';
```

Comment:
let's call this file `action-sidebar` maybe even nest it in a `sidebar/` directory.. we're likely to end up with a lot of these.